<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1426</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gismo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser_output">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>1111</width>
      <height>741</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ISOCTEUR</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'ISOCTEUR'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.25pt;&quot;&gt;a---as&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.25pt;&quot;&gt;0-----&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_serch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>161</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ISOCPEUR</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>AG1
AG5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Point list:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>check list</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_load">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load excel data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Sort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>sort list</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_getMembers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>get all points</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_Report">
       <property name="text">
        <string>GET REPORT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_full">
       <property name="text">
        <string>long report</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_info">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>190</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>App info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_print">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>150</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Print report</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_find_by_type">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>find all by types</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Type">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_minFx">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>100</y>
      <width>66</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>minFx</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_maxabsFx">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>40</y>
      <width>77</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>max|Fx|</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1296</x>
      <y>21</y>
      <width>102</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Report content:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_maxFx">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>70</y>
      <width>69</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>maxFx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>491</x>
      <y>11</y>
      <width>29</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Fx_My">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>11</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string> Fx-My</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_pltAnnot">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>127</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>plot annotation</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1426</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <action name="actionLoadXLS">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Load xls data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
